src/appendix/2-how-to-use-gdb/README.md

```diff
@@ -0,0 +1,87 @@
+# How to use GDB
+
+Below are some useful GDB commands that can help us debug our programs. This assumes you have [flashed a program](../../05-led-roulette/flash-it.md) onto your microcontroller and attached to an OpenOCD session.
+
+## General Debugging
+
+> **NOTE:** Many of the commands you see below can be executed using a short form. For example, `continue` can simply be used as `c`, or `break $location` can be used as `b $location`. While you are learning GDB try to see how short you can get the commands to go before GDB doesn't recognize it!
+
+
+### Dealing with Breakpoints
+
+* `break $location`: Set a breakpoint at a place in your code. The value of `$location` can include:
+    * `b 123` - Break on line 123 of the currently displayed file
+    * `b main.rs:123` - Break on line 123 of `main.rs`
```

Comment:
I'd say we should mention that it is possible to use break like `b function_name` or even at specific locations of memory etc, refer to the gdb help text.

Reply:
Also if we can agree to remove the shortcuts from the material for now just to make it more clear, after all they aren't being used consistently right now either, e.g. `i b` is not mentioned along `info break`.

Reply:
I added two more examples, could likely squish these into two again but have source filename, function name, label name, and line number all explained (like in help text). These would be more concise but maybe a little harder to read. Any preference?